ts/components/ui/Avatar.tsx

```diff
@@ -0,0 +1,40 @@
+// A component to provide organization logo
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+import { IOVisualCostants } from "../core/variables/IOStyles";
+import { IOColors, hexToRgba } from "../core/variables/IOColors";
+import { MultiImage } from "./MultiImage";
+
+type Avatar = {
+  logoUri: React.ComponentProps<typeof MultiImage>["source"];
+};
```

Comment:
I'm assuming that this component shouldn't have different dimensions other that 24x24. Am I right?
Would a `size` prop with `IOVisualCostants.avatarSizeDefault` as default make sense?

Reply:
We don't have any other size than 44 × 44 at the moment. But we could extend the component if we need other sizes. I prefer to keep them mapped, so no size with any number

Reply:
@mastro993 Famous last words! We recently introduced a new medium size (66 × 66) and a new rounded square container. So the new `Avatar` has `size` and `shape` props now. Already updated the branch and the above description. Let me know if there's more to debate about.

Reply:
"I know my chickens" 😆